install_prerequisites.py

```diff
@@ -35,7 +35,7 @@ def install_rvm_and_ruby():
 
 def install_fpm_gem():
     build.set_ruby_path()
-    cmd = """sudo -E su -c 'PATH="{PATH}"; export PATH; {rvm_path}/rvm reload && {rvm_path}/rvm use {ruby_version} && gem install -v 1.4.0 fpm'""".format(
+    cmd = """sudo -E su -c 'PATH="{PATH}"; export PATH; {rvm_path}/rvm reload && {rvm_path}/rvm use {ruby_version} && gem install -v 1.14.1 fpm'""".format(
```

Comment:
there was something about the version of fpm that mattered at one point... b/c version of ruby or something else...  consider a 'history' on this section of the code.

Reply:
perhaps this is what you're referring to: https://github.com/irods/externals/commit/07b68e52e6235a9feb736707b31706e87c08fb92

Reply:
good thing we don't support centos 6 anymore.

on the other hand, we can always make this specific to AlmaLinux 8.